website/src/blogs-feed.njk

```diff
@@ -0,0 +1,41 @@
+---json
```

Comment:
what's wrong with yaml?

Reply:
It's a templating file. It will generate the Yaml. That JSON is just to hold some values that are used below.

Reply:
I understand. It seems like the default for front matter in njk is yaml, not JSON, template will generate xml